resources/dicts/events/lifegen_events/murder.json

```diff
@@ -336,7 +339,8 @@
         "You tell v_c that they'll be learning how to swim today. You bring them to the river and push them in. As they paddle around uncertainly in the shallows, you suddenly grab the top of their head and force them underwater. They flail underneath your paw, exhaling bubbles and splashing wildly, but you don't let them up. Eventually, they stop moving. You let the current do its work and wash the body away."
     ],
     "murder deputy medicinecatapprentice": [
-        "You had a long argument with the leader about who v_c should be apprenticed to. Ultimately, your pleading for an apprentice was shut down. In spite of you, they even made them a medicine cat apprentice! It's odd what happens when you don't get your way. A smile of triumph spreads across your face seeing v_c bleed out helplessly. Perhaps if they were a warrior's apprentice, they could have survived."
+        "You had a long argument with the leader about who v_c should be apprenticed to. Ultimately, your pleading for an apprentice was shut down. In spite of you, they even made them a medicine cat apprentice! It's odd what happens when you don't get your way. A smile of triumph spreads across your face seeing v_c dead at your paws. Perhaps if they were a warrior's apprentice, they could have survived.",
+        "You don't understand why v_c decided to be a medicine cat apprentice instead of your own, so you decide to make them regret it, and in the morning, v_c is found dead in their nest."
```

Comment:
probably can be reworded

"You don't understand why v_c would choose to become a medicine cat apprentice, rather than your own. You decide to make sure they regret it, and come morning, v_c is found dead in their nest."